pkg/dynamicinstrumentation/diconfig/mem_config_manager.go

```diff
@@ -100,6 +100,11 @@ func (cm *ReaderConfigManager) update() error {
 		}
 
 		for pid, procInfo := range updatedState {
+			if statuses[pid] == false {
+				log.Info("Skipped the installation/deletion of probes for pid %d - failed to analyze its binary", pid)
```

Comment:
Will this be a particularly noisy log? Does it only occur when we have a probe configuration for a  proc or it's all procs that we're skipping (i.e. completely irrelevant ones)?

Reply:
It's will be as noise as the mutation of probes are. The flow is active when there's a probe configuration change, and the log is for all those processes we track (i.e processes with DI enabled). I don't think it will be that noisy, thus I'd say let's keep and and revisit it later. WDYT?

Reply:
Sounds good